### DISCUSSION TITLE
Error debug help

### DISCUSSION THREAD

**Discussion by kma33** [NREL/ReEDS-2.0#57] [Q&A: Execution] [answered]
Hi All,
I'm getting the following error when running the testing version.  Can you please provide a hint?
$eval numhintage %numhintage%
****                              $659,684
**** LINE     26 INCLUDE     D:\energy\b_inputs.gms
**** LINE      2 INPUT       D:\energy\createmodel.gms
**** 659  Error in evaluation
**** 684  Constant evaluation: BAD_CHAR

**Comment by vcelloho** [comment]
Was there a change made to b_inputs.gms, the line as it appears in github is slightly different.
https://github.com/NREL/ReEDS_OpenAccess/blob/1a59d23ac6adfff787c3038566c4eaacb42beee2/b_inputs.gms#L26

**Comment by boxblox** [comment]
Hi Katherine, this is a gams error due to a special value being passed into the $eval statement.  You will want to just double check that something strange doesn't exist in the `%numhintage%` variable.

**Comment by kma33** [comment, marked as answer]
Was originally + 2.  UPDATE: Figured it out. Thx

**Reply by vcelloho** [reply]
As written that line isn't necessary without the +2 as it sets the value equal to itself.

I agree with boxblox, check the value that numhintage is set to in the cases file that you are calling.

You can insert into GAMS code the following code to check what %numhintage% is set to prior to the line throwing an error.

$remark %numhintage%

In the listing file the output will display the value of the global variable.

GAMS 24.7.4  r58773 Released Sep 19, 2016 WEX-WEI x86 64bit/MS Windows 09/01/21 12:18:52 Page 1
G e n e r a l   A l g e b r a i c   M o d e l i n g   S y s t e m
C o m p i l a t i o n


      100

**Reply by vcelloho** [reply]
Great if you could add the solution to the issue to this thread that would be helpful for future users. Thanks!

**Reply by kma33** [reply]
Yes. No contribution here.  I didn't set up properly the cases_"whatever".csv file.
